Automaton/Features/DateWithDestiny.cs

```diff
@@ -10,9 +11,14 @@
 using FFXIVClientStructs.FFXIV.Client.Game;
 using FFXIVClientStructs.FFXIV.Client.Game.Fate;
 using FFXIVClientStructs.FFXIV.Client.Game.UI;
+using FFXIVClientStructs.FFXIV.Client.UI.Agent;
 using ImGuiNET;
 using Lumina.Excel.GeneratedSheets;
+using static Dalamud.Interface.Utility.Raii.ImRaii;
 using static ECommons.GameFunctions.ObjectFunctions;
+using static FFXIVClientStructs.FFXIV.Client.Game.UI.Telepo.Delegates;
+using static FFXIVClientStructs.FFXIV.Client.Network.NetworkModuleProxy.Delegates;
+using static IronPython.Modules._ast;
```

Comment:
lmao that's a very good question. i think visual studio decided that this was the library i needed when doing vector operations instead of system.numerics